2013/B_U2/main.cpp

```diff
@@ -0,0 +1,82 @@
+#include <iostream>
+#include <fstream>
+
+using namespace std;
+
+void skaitytiIsFailo(int &batuKiekis, int lytys[], int kojosPuse[], int dydziai[])
+{
+    ifstream data("U2.txt");
+    data >> batuKiekis;
+    
+    for (int i = 0; i < batuKiekis; i++)
+    {
+        data >> lytys[i] >> kojosPuse[i] >> dydziai[i];
+    }
+    data.close();
+}
+
+void rastiPoruKieki(int &vyriskuKiekis, int &moteriskuKiekis, int batuKiekis, int lytys[], int kojosPuse[], int dydziai[])
+{
+    int juodojoSarasoDydis = 0;
+    int juodasisSarasas[100];
+
+    for (int i = 0; i < batuKiekis; i++)
+    {
+        bool panaudotasElementas = false;
+        for (int j = 0; j < juodojoSarasoDydis; j++)
+        {
+            if (i == juodasisSarasas[j])
+            {
+                panaudotasElementas = true;
+                break;
+            }
+        }
+        if (!panaudotasElementas)
+        {
+            for (int j = i; j < batuKiekis; j++)
```

Comment:
Too much nesting. Replace it with negative if statement, or move it to separate function